src/pages/studyView/UserSelections.tsx

```diff
@@ -31,31 +30,31 @@ export default class UserSelections extends React.Component<IUserSelectionsProps
     }
 
     @computed private get clinicalDataEqualityFilters() {
-        return _.map((this.props.filter.clinicalDataEqualityFilters || []), clinicalDataEqualityFilter => {
+        return _.reduce((this.props.filter.clinicalDataEqualityFilters || []), (acc,clinicalDataEqualityFilter) => {
             let chartMeta = this.props.attributesMetaSet[clinicalDataEqualityFilter.clinicalDataType + '_' + clinicalDataEqualityFilter.attributeId];
             if (chartMeta) {
-                return (
-                    <ClinicalDataEqualityFilter
-                        chartMeta={chartMeta}
-                        values={clinicalDataEqualityFilter.values}
-                        updateClinicalDataEqualityFilter={this.props.updateClinicalDataEqualityFilter}
-                    />
-                )
-            } else {
-                return null;
+                let labelValueSet = _.reduce(clinicalDataEqualityFilter.values, (acc, next) => {
```

Comment:
no big deal but you could use _.keyBy.    i would say use _.reduce only when there's some kind of statement involved, like you're not operating for every element of input array.    you don't not need to change this.